### PR TITLE
EP-2126: fill in keywords and section by categories slugs

### PR DESCRIPTION
**Pull request by glebofff**
If the post belongs to several categories, section will be array of ~[primary_category, next_category]~ [...c1_slugs], and keywords will be merged array of ~[...rest_categories, ...tags]~ [...c2_slugs, ...c3_slugs, ..., ...tags]

Screenshot 2022-06-09 at 11 54 02

### CODE REVIEW

File: Organic/Organic.php [hunk 1]
```diff
@@ -487,18 +482,35 @@ public function getCategoryForCurrentPage() {
         }
     }
 
+    public function getSlugs( ...$terms ) {
+        return array_map(
+            fn( $term ) => $term->slug,
```
Comment (sergunich) [inline Organic/Organic.php:487]: https://www.php.net/manual/en/functions.arrow.php
> Arrow functions were introduced in PHP 7.4 as a more concise syntax for anonymous functions.

But I believe we should support PHP 7.0
https://github.com/orgnc/wordpress-plugin/blob/master/phpcs.xml#L59

@jdemaris, could you confirm?
Is the PHP 7.0 version the minimum requirement?
If yes, why this PR mentions PHP 5?
https://github.com/orgnc/wordpress-plugin/pull/72

**Review by sergunich — APPROVED** [review]
Looks good but requires a couple of fixes

And thanks again for these small refactorings/cleanups of the code!

File: Organic/Organic.php [hunk 2]
```diff
@@ -507,10 +519,19 @@ public function getTargeting() {
         $gamPageId = $gamId ? $gamId : $id;
         $gamExternalId = $id;
 
+        $firstOfRestCategories = array_shift( $restCategories );
+        $sections = $this->getSlugs( $category, $firstOfRestCategories );
```
[the remark this answers is not in the document]
Comment (glebofff) [reply Organic/Organic.php:523]: ok, makes sense

File: Organic/Organic.php (hunk 1 above)
Comment (glebofff) [reply Organic/Organic.php]: Fixed by using old syntax.

File: Organic/Organic.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (glebofff) [reply Organic/Organic.php]: done

**Review by sergunich — APPROVED** [review]

**Pull request merged**